lib/services/cordova-migration-service.ts

```diff
@@ -107,7 +107,7 @@ export class CordovaMigrationService implements ICordovaMigrationService {
 			});
 
 			let supportedPlugins = this.pluginsForVersion(toVersion).wait();
-			plugins = _.filter(plugins, plugin => _.contains(supportedPlugins, plugin));
+			plugins = _.filter(plugins, plugin => _.contains(supportedPlugins, plugin) || plugin.indexOf('@') > -1);
```

Comment:
I suggest you to check the minimum required Cordova version of the marketplace plugin and if it is above the version, that you are going to set, print Warning. In case the min supported version is below your new Cordova version, add the plugin :)